kitsune/settings.py

```diff
@@ -1037,6 +1038,10 @@ def JINJA_CONFIG():
 ]
 
 ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='', cast=Csv())
+# in production set this to 'support.mozilla.org' and all other domains will redirect.
+# can be a comma separated list of allowed domains.
+# the first in the list will be the target of redirects.
+ENFORCE_HOST = config('ENFORCE_HOST', default='', cast=Csv())
```

Comment:
I believe this should default to `None` https://github.com/dabapps/django-enforce-host/blob/master/enforce_host/__init__.py#L24

Reply:
Hmm... great catch. I'll update shortly.